### PR TITLE
Script and make target to fixup .merlin files

### PR DESCRIPTION
**Pull request by psteckler**
> Explain your changes here.

Running `make merlin-fixup` calls a script that uses `sed` to modify .merlin files, created in a Docker build, for local environment

The original .merlin file is copied as .merlin.DOCKER. Note that running the script twice will copy the modified file, so maybe this isn't so useful, or it should be named .merlin.SAVED.

> Explain how you tested your changes here.

Built locally, saved .merlin file with different name; built in Docker, ran `make merlin-fixup`; compared .merlin file with saved one, they were identical

Did that experiment for a .merlin file in src/lib/merkle_ledger_tests, and for src/app/cli/src.

Checklist:

- [ ] Tests were added for the new behavior: no
- [X] All tests pass (CI will check this if you didn't)
- [ ] Does this close issues? No

### CODE REVIEW

**Comment by bkase** [comment]
Does this work for you with your emacs setup outside of docker?

**Comment by psteckler** [comment]
> Does this work for you with your emacs setup outside of docker?

If `USEDOCKER` is not `TRUE`, then `make merlin-fixup` is a NOP, and there's a message printed to that effect.
If you run the script `merlin-fixup.sh` directly, it will have almost no effect, other than making a copy of your .merlin files to `.merlin.DOCKER`.

File: scripts/merlin-fixup.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+OPAM_SWITCH=4.07.0
+# Nominal OCaml version in the Docker container
+OCAML_VERSION=4.07
+
+for file in `find src -name .merlin`
+do
+    cp -p $file $file.DOCKER
+    sed --in-place s+.opam/$OCAML_VERSION/+.opam/$OPAM_SWITCH/+g $file
+    sed --in-place s+/home/opam/.opam/+$HOME/.opam/+g $file
+    sed --in-place s+/home/opam/app/src/_build/+$(git-root)/src/_build/+g $file
```
Comment (jkrauska) [inline scripts/merlin-fixup.sh]: check for git-root earlier on and exit if its not present?

Comment (psteckler) [reply scripts/merlin-fixup.sh]: Changed to use `git rev-parse`.

**Pull request merged**